functions/fisher_uninstall.fish

```diff
@@ -111,7 +111,7 @@ function fisher_uninstall -d "Disable / Uninstall Plugins"
             continue
         end
 
-        set -l tmp (mktemp)
+        set -l tmp (mktemp -t fisher.XXXXXXXX)
```

Comment:
You are right. I have fixed it in 0.2.0 which I should be pushing shortly. :smile: